packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -405,6 +410,26 @@ public void updateClippingRect() {
     updateClippingToRect(mClippingRect);
   }
 
+  @Override
+  public void startViewTransition(View view) {
+    super.startViewTransition(view);
+    mIsTransitioning = true;
+  }
+
+  @Override
+  public void endViewTransition(View view) {
+    super.endViewTransition(view);
+    mIsTransitioning = false;
+    mChildrenRemovedWhileTransitioning = null;
+  }
+
+  public Set<Integer> ensureChildrenRemovedWhileTransitioning() {
```

Comment:
Can this be private?

Reply:
https://github.com/facebook/react-native/pull/47634/commits/c14e0268fcc10a78bb60fa65a274f34b14e808c2